src/main/kotlin/com/sourcegraph/cody/edit/EditService.kt

```diff
@@ -74,8 +70,11 @@ class EditService(val project: Project) {
       // TODO: We need to support the file-level operations.
       when (op) {
         is CreateFileOperation -> {
-          logger.warn("Workspace edit operation created a file: ${op.uri}")
-          return false
+          logger.info("Workspace edit operation created a file: ${op.uri}")
+          val file =
+              CodyEditorUtil.createFileOrScratchFromUntitled(project, op.uri, content = "")
+                  ?: return false
+          CodyEditorUtil.showDocument(project, file)
```

Comment:
Should create function really show a file? Isn't that a separate call from agent?